modernizer-maven-plugin/src/main/java/org/gaul/modernizer_maven_plugin/Modernizer.java

```diff
@@ -250,6 +273,23 @@ private boolean ignoreClass() {
         return false;
     }
 
+    private boolean ignoreMethod(String methodName, String methodDescriptor) {
+        String returnType = Type.getReturnType(methodDescriptor).getClassName();
+        StringBuilder arguments = new StringBuilder("");
+        for (Type arg : Type.getArgumentTypes(methodDescriptor)) {
+            arguments.append(" ");
+            arguments.append(arg.getClassName());
+        }
+        String methodDescription =
+            className + " " + methodName + " " +
+            (returnType + " " + arguments.toString().trim())
+            .replace('$', '.');
```

Comment:
I think this will apply to `(returnType + " " + arguments.toString().trim())` rather than the entire description.

Reply:
Yeah. The intention is to replace `$` with `.` only in 
`(returnType + " " + arguments.toString().trim())` . the `className` part of the description should have `$` characters in it.